staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/value_test.go

```diff
@@ -322,9 +322,9 @@ func TestMapValueIsSet(t *testing.T) {
 
```

Comment:
revert changes to this file?

Reply:
reverted, these changes are from the update vendor (to bring in otel pkgs) I _think_